spec/integration.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+# ------------------------------------- #
+#  cocoapods-bazel Integration tests    #
+# ------------------------------------- #
+
+#-----------------------------------------------------------------------------#
+
+# The following integrations tests are based on file comparison.
+#
+# 1.  For each test there is a folder with a `before` and `after` subfolders.
+# 2.  The contents of the before folder are copied to the `TMP_DIR` folder and
+#     then the given arguments are passed to the `POD_BINARY`.
+# 3.  After the pod command completes the execution the each file in the
+#     `after` subfolder is compared to be to the contents of the temporary
+#     directory.  If the contents of the file do not match an error is
+#     registered. Xcode projects are compared in an UUID agnostic way.
+#
+# Notes:
+#
+# - The output of the pod command is saved in the `execution_output.txt` file
+#   which should be added to the `after` folder to test the CocoaPods UI.
+# - To create a new test, just create a before folder with the environment to
+#   test, copy it to the after folder and run the tested pod command inside.
+#   Then just add the tests below this files with the name of the folder and
```

Comment:
Don't need to add paths anywhere, right? Looks like this is handled with `spec_dir.join('integration').each_child`

Reply:
Yup, from 2 copy-pastes ago 🙈